src/tests/blueprint/t_blueprint_mesh_relay.cpp

```diff
@@ -845,3 +845,45 @@ TEST(conduit_blueprint_mesh_relay, sparse_topos)
 }
 
 
+
+//-----------------------------------------------------------------------------
+TEST(conduit_blueprint_mesh_relay, spiral_multi_file_yaml_json_hdf5)
+{
+    Node io_protos;
+    relay::io::about(io_protos["io"]);
+    bool hdf5_enabled = io_protos["io/protocols/hdf5"].as_string() == "enabled";
+    bool silo_enabled = io_protos["io/protocols/conduit_silo_mesh"].as_string() == "enabled";
+    Node data;
+
+    // use spiral , with 7 domains
+    conduit::blueprint::mesh::examples::spiral(7,data);
+
+    std::vector<std::string> protocols = {"yaml","json"};
+    if(hdf5_enabled)
+    {
+        protocols.push_back("hdf5");
+    }
+
+    // TODO: finish wire up and we can add this case
+    // if(silo_enabled)
+    // {
+    //     protocols.push_back("silo");
+    // }
```

Comment:
TODO?

Reply:
not fully ready yet.

This adds logic to detect silo,but it doens't wire up the blueprint write or read yet.

Reply:
this is now wired up